README.md

```diff
@@ -27,6 +30,12 @@ Find another group to work with. The other group should be students who neither
 (7 minutes) The Designer's first step in design thinking is empathizing with the person who the design is for.  The Designer should have a conversation with the Customer to get to know them and to ask about the Customer's needs and interests.  The goal is to try to identify the customer's need, *not a specific solution*.  The Designers who are going first should interview the customer and try to figure out the underlying need they'd like to satisfy.  
 
 ***Q1.2 Share some notes from your interview here and try to describe the "need" with a very brief description:***
+##### Answer:
+| Feature   |      Description      |
+|----------|:-------------:|
+| light Weight|  When students walk to the campus, it is difficult to carry heavy stuff  |
```

Comment:
Great list of features. In particular, your point about safety is relevant and balancing that with entertainment in a device is something we should give thought to in the present and in the future.